docs/source/404.md

```diff
@@ -28,4 +28,4 @@ do so we have moved some files around and we have
 modified the base url in order to support multi version docs
 (you'll see the version switcher at the bottom right of the page).
 
-Back to {{ '[documentation homepage](https://docs.pymc.io/en/{}/)'.format(version_slug) }}
```

Comment:
the 404 file is special and should html link, we should exclude this file from the pre-commit hook.

Reply:
It's now how it was before